example_configs/mailserver.md

```diff
@@ -0,0 +1,127 @@
+# Mailserver Docker
+
+[Docker-mailserver](https://docker-mailserver.github.io/docker-mailserver/latest/) is a Production-ready fullstack but simple mail server (SMTP, IMAP, LDAP, Antispam, Antivirus, etc.) running inside a container.
+
+To integrate with LLDAP, make sure you fill in the attributes correctly
+
+Exemple starting [docker-compose](https://github.com/docker-mailserver/docker-mailserver/blob/master/compose.yaml)
+
+Most important part :
+```yaml
+      # >>> Postfix LDAP Integration
+      - ACCOUNT_PROVISIONER=LDAP
+      - LDAP_SERVER_HOST=lldap:3890
+      - LDAP_SEARCH_BASE=dc=d3n,dc=com
+      - LDAP_BIND_DN=uid=admin,ou=people,dc=d3n,dc=com
+      - LDAP_BIND_PW=admin123$*
+      - LDAP_QUERY_FILTER_USER=(&(objectClass=inetOrgPerson)(|(uid=%u)(mail=%u)))
+      - LDAP_QUERY_FILTER_GROUP=(&(objectClass=groupOfUniqueNames)(|(uid=%s)(cn=%s)))
```

Comment:
for LLDAP, uid and cn are the same thing, you can simplify the query to just uid